validator/errorbundler.py

```diff
@@ -280,18 +283,25 @@ def _merge_messages(self, messages, callback, name):
             else:
                 trace.append(message["file"])
 
+            keys = ("column",
+                    "compatibility_type",
+                    "context",
+                    "description",
+                    "editors_only",
+                    "for_appversions",
+                    "line",
+                    "signing_severity",
+                    "tier")
+
+            kw = {key: message[key] for key in keys if key in message}
+
```

Comment:
Ah, right, and the other issue was that the arguments actually need to be left out if they're not in the original message. So I needed to switch to a dict of kwargs either way, and add conditionals for at least those two properties. Just handling all of the properties the same way seemed like the cleaner solution.